examples/example_plugin/docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Jinja2==3.0.3
+mkdocs==1.3.0
+mkdocs-include-markdown-plugin==3.0.1
```

Comment:
Looks like we have `mkdocs-include-markdown-plugin = "~3.2.3"` in pyproject.toml, would be good to have these match versions.

Reply:
I was trying to match what core added in next, but didn't update the pyproject.toml

Reply:
Oh, I found a bug in Nautobot then, the pyproject.toml does not match the docs/requirements.txt

Reply:
🤷 I think it was just not in the scope of this PR: https://github.com/nautobot/nautobot/pull/1314/files but we probably should take a look at bumping those.

Reply:
Bumped.